### PR TITLE
EX-1503 - Style the sicherearbeit.at download and download list

### PR DESCRIPTION
**Pull request by Bejor**
Bildschirmfoto 2021-09-20 um 18 58 26
Bildschirmfoto 2021-09-20 um 18 58 22

### CODE REVIEW

File: src/components/ex-downloadlist/ex-downloadlist.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      @apply text-l font-semibold uppercase;
+    }
+
+    .downloadlist__subtitle {
+      @apply text-s mt-2;
+    }
+  }
+
+  .downloadlist__elements {
+    @apply flex flex-col pl-0 mt-4 mb-0 list-none;
```
Comment (martinkastler) [inline src/components/ex-downloadlist/ex-downloadlist.scss:36]: There is a padding-bottom on the ul that can be removed as it makes the y-spaces look uneven:

image

File: src/components/ex-download/ex-download.scss [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    background-color: rgba(65, 131, 215, 1);
+
+    &:before {
+      @apply mr-2;
+      content: '\26A0';
+    }
+  }
+
+  &_item {
+    @apply flex flex-col w-full px-4 py-2;
```
Comment (martinkastler) [inline src/components/ex-download/ex-download.scss:15]: The single download item looks better having .p-4. But then the download item within the list inflate as well. In ideal case a single download has .p-4 and downloads within the download list remain having .px-4.py-3

File: src/components/ex-download/ex-download.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      }
+    }
+  }
+  &_description {
+    @apply hidden;
+  }
+  &_button {
+    @apply hidden;
+  }
+  &.download-preview {
```
Comment (martinkastler) [inline src/components/ex-download/ex-download.scss:46]: The gray does not match the one of the other boxes. Same goes for font colors I believe and definitely the underlines.

image

**Review by martinkastler — CHANGES_REQUESTED** [review]

File: src/components/ex-downloadlist/ex-downloadlist.scss (hunk 1 above)
Comment (Bejor) [reply src/components/ex-downloadlist/ex-downloadlist.scss]: Removed the space

File: src/components/ex-download/ex-download.scss (hunk 2 above)
Comment (Bejor) [reply src/components/ex-download/ex-download.scss]: changed the spacing

File: src/components/ex-download/ex-download.scss (hunk 3 above)
Comment (Bejor) [reply src/components/ex-download/ex-download.scss:50]: changed the color

File: src/components/ex-download/ex-download.scss [hunk 4]
```diff
@@ -45,28 +49,35 @@
   }
   &.download-preview {
     .download_item {
-      @apply bg-gray-3 pb-6;
+      @apply pb-6;
```
Comment (martinkastler) [inline src/components/ex-download/ex-download.scss:52]: Turn .pb-6 to .py-6 to make it awesome.

File: src/components/ex-download/ex-download.scss [hunk 5]
```diff
@@ -45,28 +49,35 @@
...
         .download_image {
           @apply inline-flex;
           max-width: 200px;
         }
         .download_title {
-          @apply text-s items-center justify-center w-full mt-2 mb-0 font-semibold no-underline;
+          @apply text-s text-gray-1 items-center justify-center w-full mt-2 mb-0 font-semibold no-underline;
+
+          &:hover {
+            @apply text-primary;
```
Comment (martinkastler) [inline src/components/ex-download/ex-download.scss:68]: Something is wrong with the underline. The underline stays gray on mouse-over. Should also turn blue:

image

Comment (Bejor) [reply src/components/ex-download/ex-download.scss:78]: Should be fixed now

File: src/components/ex-download/ex-download.scss (hunk 4 above)
Comment (Bejor) [reply src/components/ex-download/ex-download.scss:56]: changed it

**Review by martinkastler — APPROVED** [review]

**Pull request merged**